### DISCUSSION TITLE
HANNIBAL TV tunisia

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#770] [Q&A]
### Channel Name

HANNIBAL TV

### Channel ID

HannibalTV.tn

### Website

https://ok.ru/live/3724068265546

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
OK.ru links cannot be used in this case, as their tokens are IP-locked.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19758
